subset/security/Dockerfile.test_password

```diff
@@ -0,0 +1,15 @@
+FROM daq/aardvark:latest
+
+RUN $AG update && $AG install openjdk-8-jre
+
+RUN $AG update && $AG install openjdk-8-jdk git
+
+RUN $AG update && $AG install curl 
+
+COPY subset/security/ .
+
+RUN curl https://svn.nmap.org/nmap/nmap-mac-prefixes | tee brute_test/src/main/resources/macList.txt
```

Comment:
In the future, I feel like this macList.txt curl would be better placed in a shared function that all tests can access

Reply:
unfortunately getting the file on the build of the dockerfile is doing exactly that for that only instance of dockerfile